core/src/com/unciv/logic/map/MapUnit.kt

```diff
@@ -961,7 +963,12 @@ class MapUnit {
     }
 
     fun getPressureAddedFromSpread(): Int {
-        return baseUnit.religiousStrength
+        var pressureAdded = baseUnit.religiousStrength.toFloat()
+        for (unique in civInfo.getMatchingUniques("[]% Spread Religion Strength for [] units"))
+            if (matchesFilter(unique.params[0]))
+                pressureAdded *= 1f + unique.params[0].toFloat() / 100f
```

Comment:
Maybe we should add a String.toPercent since we use this combo all the time.
For another day...